frontend/src/main/scala/bloop/dap/DebugServer.scala

```diff
@@ -17,6 +18,10 @@ final class StartedDebugServer(
 )
 
 object DebugServer {
+  // code evaluation is not supported, so `toString` methods
+  // must not be called for classes implementing them
```

Comment:
Please use `/* */`-style comments and elaborate more on what's happening here. Why does `toString` force the evaluation of code? It's not obvious that these are the semantics of java-debug, so a more complete comment explaining all parts involved would help motivate the use of this side-effecting function in the constructor.